src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnectionX.cs

```diff
@@ -32,8 +32,15 @@ internal sealed class SqlConnectionX : DbConnection, ICloneable
         private SqlDataSource? _dataSource;
         private SqlConnector? _internalConnection;
 
+        bool _disposed;
+
+        internal SqlConnector? InternalConnection { get => _internalConnection; }
```

Comment:
I honestly forget, is this the same as `InternalConnection => _internalConnection;`? I forget if that does set and get or if it just does get.

Reply:
Also, properties should go below constructors

Reply:
Looks like it's equivalent: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-properties#the-get-accessor